var/spack/repos/builtin/packages/charliecloud/package.py

```diff
@@ -101,14 +106,18 @@ class Charliecloud(AutotoolsPackage):
         sha256="15ce63353afe1fc6bcc10979496a54fcd5628f997cb13c827c9fc7afb795bdc5",
     )
     variant("docs", default=False, description="Build man pages and html docs")
-    variant("squashfuse", default=False, description="Build with squashfuse support")
-
+    variant(
+        "squashfuse", default=False, description="Build with squashfuse support", when="@0.32:"
+    )
     # Autoconf.
     depends_on("m4", type="build")
     depends_on("autoconf", type="build")
     depends_on("automake", type="build")
     depends_on("libtool", type="build")
 
+    # pkg-config for 0.36.
+    depends_on("pkg-config", type="build", when="@0.36")
```

Comment:
Looks like this should probably be

```
depends_on("pkg-config", type="build", when="@0.36")
depends_on("pkg-config", type="build", when="+squashfuse")
```

we shouldn't rely on it coincidentally being available from a dependency -- if the build uses it directly, create a direct dependency.

Reply:
We need to keep the dependency on `pkg-config` for 0.36 because we erroneously require it regardless of the squashfuse setting. I hope to remove this in 0.37, where it will only be used if squashfuse is enabled. (Note that the default in our `configure` is to enable squashfuse.)